docker-storage-setup.service

```diff
@@ -5,7 +5,7 @@ Before=docker.service
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/docker-storage-setup /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage
+ExecStart=/usr/bin/container-storage-setup /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage
 EnvironmentFile=-/etc/sysconfig/docker-storage-setup
```

Comment:
I don't think we should call it container-storage-setup.service.  I would rather remove it all together or use it as an example of how you could use a config file.